### ISSUE TITLE
Results of searching on module and searching manually are different

### ISSUE THREAD

**Issue by redrussianarmy** [bisoncorps/search-engine-parser#98]
Results of searching on module and searching manually in any web browser are different. For instance, I want to search on google with filtering of wikipedia (e.g. keyword site:tr.wikipedia.org) and scrape first result. Generally, a first result is useless for me when tried to search on module. I am using following class and its "run" method:

```
from search_engine_parser.core.engines.google import Search as GoogleSearch
class SearchOnline():
    def __init__(self):
        self.target_url = ""
    def run(self, keyword):
        search_args = ((keyword + ' site:tr.wikipedia.org'), 1)
        gsearch = GoogleSearch()
        gresults = gsearch.search(*search_args, url="google.com.tr")
        self.target_url = gresults["links"][0]
        return self.target_url
```

**Here you can find following scenario:**
('istanbul site:tr.wikipedia.org', 1)   -> _These are my searching args in code_

_---- These are results (I take 0th url) ----_
['https://tr.wikipedia.org/wiki/Portal:%C4%B0stanbul', 'https://tr.wikipedia.org/wiki/%C4%B0stanbul_Vilayeti', 'https://tr.wikipedia.org/wiki/%C4%B0stanbul%27daki_meydanlar_listesi', 'https://tr.wikipedia.org/wiki/%C4%B0stanbul_tarihi', 'https://tr.wikipedia.org/wiki/Kategori:%C4%B0stanbul%27un_b%C3%B6lgeleri', 'https://tr.wikipedia.org/wiki/%C4%B0stanbul%27un_il%C3%A7eleri', 'https://tr.wikipedia.org/wiki/%C4%B0stanbul%27un_idari_yap%C4%B1lanmas%C4%B1', 'https://tr.wikipedia.org/wiki/Konstantinopolis', 'https://tr.wikipedia.org/wiki/Kategori:%C4%B0stanbul', 'https://tr.wikipedia.org/wiki/Be%C5%9Fikta%C5%9F,_%C4%B0stanbul']

_---- This is result of manual searching (It is not even in results array of module) ----_
https://tr.wikipedia.org/wiki/%C4%B0stanbul

**Here is screenshot of manual searching results.**
![Screenshot_issue](https://user-images.githubusercontent.com/24498747/87764631-5e4f0600-c81f-11ea-8042-44c9a2de258f.png)

**Desktop:**
 - OS: Ubuntu 18.04.4 LTS
 - Python Version: 3.6.9
 - Search-engine-parser version: 0.6

**Comment by deven96** [comment]
Solved by the version 0.6.1
Please @all-contributors Add @redrussianarmy for contribution

**Comment by deven96** [comment]
Solved by the version 0.6.1
Please @all-contributors Add @redrussianarmy for contribution bug report
